README.md

```diff
@@ -12,7 +12,20 @@
 $ sh ./gradlew -Pjdk18_home=<jdk18_home_dir> -Pllvm_home=<libclang_dir> clean verify
 ```
 
-After building, there should be a new `jextract` folder under `build` (the contents and the name of this folder might vary slightly depending on the platform):
+<details><summary>Using a local installation of LLVM</summary>
```

Comment:
Can we do this in bold, and also perhaps use the quote mechanism - e.g.

> <details><summary><b>Using a local installation of LLVM</b></summary>Some text</summary>

Reply:
> Can we do this in bold, and also perhaps use the quote mechanism - e.g.
> 
> > Using a local installation of LLVMSome text

The above was done like this:

```
> <details><summary><b>Using a local installation of LLVM</b></summary>Some text</summary>
```

Reply:
Done